python/lsst/ctrl/bps/wms/panda/cmd_line_embedder.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+
+class CommandLineEmbedder:
+    """
+    Class embeds static (constant across a task) values into the pipeline execution command line
+    and resolves submission side environment variables
+
+    Parameters
+    ----------
+    config : `~lsst.ctrl.bps.BPSConfig`
+            BPS configuration that includes the list of dynamic (uniques per job)
+            and submission side resolved variables
+
+    """
+    def __init__(self, config):
+        self.leave_placeholder_params = config.get("placeholderParams", ['qgraphNodeId', 'qgraphId'])
+        self.submit_side_resolved = config.get("submitSideResolvedParams", ['USER'])

```

Comment:
I think we should just change the submission yaml examples to use 'operator' instead of 'user' to solve this problem.